spec/models/commodity_spec.rb

```diff
@@ -188,21 +248,21 @@
       let(:commodity)    { create :commodity, :with_indent, validity_start_date: 3.years.ago.beginning_of_day }
       let!(:measure1)    do
         create :measure, :with_base_regulation, measure_sid: 1,
-                         measure_type_id: measure_type.measure_type_id,
-                         additional_code_type_id: nil,
-                         goods_nomenclature_sid: commodity.goods_nomenclature_sid,
-                         validity_start_date: 1.year.ago.beginning_of_day
+                                                measure_type_id: measure_type.measure_type_id,
+                                                additional_code_type_id: nil,
+                                                goods_nomenclature_sid: commodity.goods_nomenclature_sid,
+                                                validity_start_date: 1.year.ago.beginning_of_day
```

Comment:
I am not sure if this indentation is better than the previous one,
has Rubocop complained about it?

Reply:
Yeah